app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java

```diff
@@ -116,6 +124,19 @@ void testAutomatedClaim_missingAnchor() {
         masResponse.getMessage());
   }
 
+  @Test
+  void testPdfGenerationAndFetch() throws Exception {
+    TestSetup setup = TestSetup.getInstance("test-7101-01");
+    helper.setApiKey("test-key-01");
+    ResponseEntity<String> response = helper.generatePdf(setup);
+    Assertions.assertEquals(HttpStatus.OK, response.getStatusCode());
+    String actual = response.getBody();
+    String expected = setup.getGeneratePdfResponse();
+    JSONAssert.assertEquals(expected, actual, JSONCompareMode.STRICT);
+    ResponseEntity<byte[]> fetchResponse = helper.getPdf(setup);
+    Assertions.assertEquals(HttpStatus.OK, fetchResponse.getStatusCode());
```

Comment:
I was imagining this assertion as part of the `testAutomatedClaim` test. That endpoint should kick off the process that leads to generating a pdf. For end2end testing we want to make sure that it actually gets to that point in the  process.

Reply:
You may need to add in an await function to give it a minute for the processing to go through